sandbox/modules/identity/src/main/java/org/opensearch/identity/ConfigConstants.java

```diff
@@ -15,6 +15,15 @@ public class ConfigConstants {
     public static final String IDENTITY_CONFIG_INDEX_NAME = "identity.config_index_name";
     public static final String IDENTITY_AUTH_MANAGER_CLASS = "identity.auth_manager_class";
     public static final String IDENTITY_ALLOW_DEFAULT_INIT_SECURITYINDEX = "plugins.identity.allow_default_init_securityindex";
-
     public static final String IDENTITY_ENABLED = "identity.enabled";
+
+    // REST Action and API
+    public static final String IDENTITY_REST_REQUEST_PREFIX = "_identity";
```

Comment:
Should we have two separate constant files. I'm inclined towards keeping only one constant file.

Reply:
I think the action names should be declared in the `Action` classes themselves and use their full string to make them easy to search through the whole code base. When I do a full codebase search for `indices:data/read/search` it's able to find the corresponding action.

This class is called `ConfigConstants` because it mostly contains values that would be placed in `opensearch.yml`, but it's become a dumping ground for constants. I would be in favor of a `IdentityRestConstants` or just `RestConstants` in a `rest` package to keep them closer to the API actions. Long-term there should be a clear distinction of where to place constants.

Reply:
For whatever it is worth, I am planning on having RestConstants inside the rest package if you wanted to go that route too.

Reply:
okay.. makes sense. I have not transformed action names into constant variables, so for this PR `cluster:admin/user/create` should be searchable.

I will create a new constants file and update the description of ConfigConstants to reflect that this is only for variables to be used in opensearch.yml